ward/collect.py

```diff
@@ -13,18 +13,22 @@ def is_test_module(module: pkgutil.ModuleInfo) -> bool:
     return module.name.startswith("test_")
 
 
-def get_info_for_modules(path: str) -> Generator[pkgutil.ModuleInfo, None, None]:
-    # Check for modules at the root of the specified path
-    for module in pkgutil.iter_modules([path]):
+def get_info_for_modules(path: List[str]) -> Generator[pkgutil.ModuleInfo, None, None]:
```

Comment:
Small nitpick: you could update the param name from `path` to `paths` since multiple paths are now supplied.